configs/manager/halfstack.toml

```diff
@@ -21,6 +21,7 @@ service-addr = { host = "0.0.0.0", port = 8081 }
 ssl-enabled = false
 #ssl-cert = "/etc/backend.ai/ssl/apiserver-fullchain.pem"   # env: BACKNED_SSL_CERT
 #ssl-privkey = "/etc/backend.ai/ssl/apiserver-privkey.pem"  # env: BACKNED_SSL_KEY
+enabled-session-managing-contexts = { "PREPARING" = "1h", "TERMINATING" = "30m" }
```

Comment:
Please write down briefly about this value (what this value means, available values, ...) on `configs/manager/sample.toml` and define scheme on manager's config.py

Reply:
@kyujin-cho Thanks for the feedback!
https://github.com/lablup/backend.ai/blob/632ca06e46f966f24f87805b66118207e8bd4eda/configs/manager/sample.toml#L116-L119